fcrepo-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -34,6 +34,16 @@
 
 	</servlet-mapping>
 
+  <filter>
+    <filter-name>ETagFilter</filter-name>
+    <filter-class>org.springframework.web.filter.ShallowEtagHeaderFilter</filter-class>
+  </filter>
+
+  <filter-mapping>
+    <filter-name>ETagFilter</filter-name>
+    <url-pattern>/static/*</url-pattern>
```

Comment:
This already exists and is used, e.g. by the HTML UI and also for delivering static RDF related to constraint violations. This change simply adds ETags to those responses.